jupyter-book/spatial/imputation.yml

```diff
@@ -0,0 +1,11 @@
+name: imputation
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+- python>=3.9
```

Comment:
This is dangerous since many packages aren't compatible with e.g. Python 3.11. I'd pin this explicitly.